AdminCore.WebApp/Controllers/TeamController.cs

```diff
@@ -46,6 +47,20 @@ public IActionResult GetTeamById(int id)
       return StatusCode((int)HttpStatusCode.InternalServerError, $"No team found with an ID of { id.ToString() }");
     }
 
+    [HttpGet("getTeamsByProjectId/{projectId}")]
+    [ProducesResponseType(typeof(IList<TeamViewModel>), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    public IActionResult GetTeamsByProjectById(int projectId)
```

Comment:
I don't see any tests that test this function

Reply:
AdminCore.Services.Tests/TeamServiceTests.cs https://github.com/UnosquareBelfast/admin-backend/blob/bf2267c6be093020c70e1430073eeb8d8d83e0ee/AdminCore.Services.Tests/TeamServiceTests.cs

Reply:
Ah sorry i misread

Reply:
Added tests